usage.txt

```diff
@@ -1,13 +1,18 @@
 
-  0x [flags] file.js
+  0x [flags] -- node [nodeFlags] script.js [scriptFlags]
```

Comment:
Can you make the `0x` part in this document customizable? It will help for integration purposes.

Reply:
added a customisable banner option